submissions/Nemooochka/html-css-popup/index.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <title>Html-css popup</title>
+
+    <script src="https://kit.fontawesome.com/559af75b05.js" crossorigin="anonymous"></script>
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
```

Comment:
It's not a blocker, but you can use some simpler styles normalizer but without external links for them.